### PR TITLE
Implement vmotion test cases[skip ci]

### LINKED ISSUE #2253: Execute the vMotion test cases
Execute the test cases defined in #2250 on Nimbus cluster

### PR DESCRIPTION
**Pull request by mhagen-vmware**
Fixes #2253

### CODE REVIEW

File: tests/manual-test-cases/Group13-vMotion/13-1-vMotion-VCH-Appliance.robot [hunk 1]
```diff
@@ -0,0 +1,132 @@
+*** Settings ***
+Documentation  Test 13-1 - vMotion VCH Appliance
+Resource  ../../resources/Util.robot
+Suite Setup  Create a VSAN Cluster
+Suite Teardown  Run Keyword And Ignore Error  Kill Nimbus Server  %{NIMBUS_USER}  %{NIMBUS_PASSWORD}  *
```
Comment (rajanashok) [inline tests/manual-test-cases/Group13-vMotion/13-1-vMotion-VCH-Appliance.robot]: Not sure if the container log collection is part of the Teardown yet, its a nice to have during our nightly run log upload process.

Comment (mhagen-vmware) [inline tests/manual-test-cases/Group13-vMotion/13-1-vMotion-VCH-Appliance.robot]: container log collection is handled in the test cases under the cleanup vic machine keyword

**Comment by matthewavery** [comment]
LGTM

**Comment by rajanashok** [comment]
LGTM

**Comment by mhagen-vmware** [comment]
.

**Pull request merged**